src/main/steps/common/page.njk

```diff
@@ -8,10 +8,12 @@
   <!-- {{ formState.id }} -->
 {% endblock %}
 
+{% set serviceName = (applyForDivorce if isDivorce else applyForDissolution) | capitalize %}
```

Comment:
Shouldn't this just be a variable set in the common content?

Reply:
👍 Good idea, I've moved this out to common content in 6f7690a